Library/Homebrew/cmd/search.rb

```diff
@@ -141,13 +141,20 @@ def search_tap(user, repo, rx)
     []
   end
 
-  def search_formulae(rx)
+  # @param [Regexp] rx
+  # @param [Hash] options
+  # @option options [Boolean] :installed If true, include installed formulae
```

Comment:
By not highlight, I mean not appending `(installed)`. Also current highlight function inside `puts_columns` is kinda of inefficient, for its unnecessary item replacement.